src/test/java/org/folio/rest/impl/ApiTestBase.java

```diff
@@ -17,22 +17,20 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.Map;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 import java.util.stream.Stream;
 
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 import static org.folio.rest.RestVerticle.OKAPI_HEADER_TENANT;
 import static org.folio.rest.RestVerticle.OKAPI_HEADER_TOKEN;
-import static org.folio.rest.impl.AbstractHelper.ID;
 import static org.folio.rest.impl.AbstractHelper.OKAPI_URL;
 import static org.folio.rest.impl.ApiTestSuite.mockPort;
 import static org.junit.Assert.fail;
 
 public class ApiTestBase {
 
-  static final String UUID = "8d3881f6-dd93-46f0-b29d-1c36bdb5c9f9";
+  static final String _UUID = "8d3881f6-dd93-46f0-b29d-1c36bdb5c9f9";
```

Comment:
Is underscore added intentionally?

Reply:
yes, because of naming conflict with UUID class from java.util package